README.md

```diff
@@ -183,6 +183,60 @@ module.exports = {
 };
 ```
 
+### With swc-loader
+
+Your `@swc/core` version must be at least 1.2.52 and `swc-loader` version greater than 0.1.13 is recommened.
+
+> ⚠ `@swc/core` and `swc-loader` are maintained by the community not by Facebook.
+
+```js
+const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
+const webpack = require('webpack');
+const ReactRefreshTypeScript = require('react-refresh-typescript');
```

Comment:
Not an approval @Austaras 
But QQ. is `react-refresh-typescript` something that's needed for fast-refresh to work?

Reply:
My mistake. Fixed

Reply:
Thanks :D